### PR TITLE
Prepare for 2.1.0 "Olive Tin Hamster"

### PR DESCRIPTION
**Pull request by marissaDubbelaar**
### `Added`

- Inclusion of assets/schema_input.json
- Added the multiQC again to report the versions
- MHCquant parameters are now directly assigned to the argument of the process

### `Fixed`

- Fixed typos
- #165 
- #176 

### `Dependencies`

Note, since the pipeline is now using Nextflow DSL2, each process will be run with its own [Biocontainer](https://biocontainers.pro/#/registry). This means that on occasion it is entirely possible for the pipeline to be using different versions of the same tool. However, the overall software dependency changes compared to the last release have been listed below for reference.

| Dependency            | Old version | New version |
| --------------------- | ----------- | ----------- |
| `openms`              | 2.5.0       | 2.6.0       |
| `openms-thirdparty`   | 2.5.0       | 2.6.0       |
| `thermorawfileparser` | 1.2.3       | 1.3.4       |

## PR checklist

- [X] This comment contains a description of changes (with reason).
- [X] Make sure your code lints (`nf-core lint`).
- [X] Ensure the test suite passes (`nextflow run . -profile test,docker`).
- [X] Usage Documentation in `docs/usage.md` is updated.
- [X] Output Documentation in `docs/output.md` is updated.
- [X] `CHANGELOG.md` is updated.
- [X] `README.md` is updated (including new tool citations and authors/contributors).

### CODE REVIEW

**Review by Leon-Bichmann — APPROVED** [review]
I only skimmed through it quickly and it looks good to me, no changes required from my side.

File: modules/local/openms_thermorawfileparser.nf [hunk 1]
```diff
@@ -30,6 +30,11 @@ process OPENMS_THERMORAWFILEPARSER {
             ThermoRawFileParser.sh -i=${rawfile} \\
                 -f=2 \\
                 -b=${prefix}.mzML
-            ThermoRawFileParser.sh --version &> ThermoRawFileParser.version.txt
+            > ThermoRawFileParser.version.txt
```
Comment (ggabernet) [inline modules/local/openms_thermorawfileparser.nf:33]:
```suggestion
```

Comment (ggabernet) [reply modules/local/openms_thermorawfileparser.nf:33]: Not sure but think this should be removed here

File: modules/nf-core/modules/fastqc/functions.nf [hunk 2]
```diff
@@ -0,0 +1,78 @@
+//
```
Comment (ggabernet) [inline modules/nf-core/modules/fastqc/functions.nf:1]: Not used, can be removed

File: modules/nf-core/modules/fastqc/main.nf [hunk 3]
```diff
@@ -0,0 +1,54 @@
+// Import generic module functions
```
Comment (ggabernet) [inline modules/nf-core/modules/fastqc/main.nf:1]: Not used, can be removed

File: modules/nf-core/modules/fastqc/meta.yml [hunk 4]
```diff
@@ -0,0 +1,52 @@
+name: fastqc
```
Comment (ggabernet) [inline modules/nf-core/modules/fastqc/meta.yml:1]: Not used, can be removed

File: nextflow.config [hunk 5]
```diff
@@ -181,29 +194,29 @@ process.shell = ['/bin/bash', '-euo', 'pipefail']
...
 manifest {
-    name = 'nf-core/mhcquant'
-    author = 'Leon Bichmann'
-    homePage = 'https://github.com/nf-core/mhcquant'
-    description = 'Identify and quantify peptides from mass spectrometry raw data'
-    mainScript = 'main.nf'
-    nextflowVersion = '>=21.04.0'
-    version = '2.0.0'
+    name            = 'nf-core/mhcquant'
+    author          = 'Leon Bichmann'
```
Comment (ggabernet) [inline nextflow.config:214]:
```suggestion
    author          = 'Leon Bichmann, Marissa Dubbelaar'
```

File: workflows/mhcquant.nf [hunk 6]
```diff
@@ -193,6 +193,7 @@ workflow MHCQUANT {
             .map(it -> [it[1], it[2], it[3]])
         // If specified translate variants to proteins and include in reference fasta
         GENERATE_PROTEINS_FROM_VCF( ch_vcf )
+        // ch_versions = ch_versions.mix(GENERATE_PROTEINS_FROM_VCF.out.versions.first().ifEmpty(null))
```
Comment (ggabernet) [inline workflows/mhcquant.nf:196]: Should the versions be added here?

**Review by ggabernet — COMMENTED** [review]
Looking good, just had some minor comments

File: workflows/mhcquant.nf (hunk 6 above)
Comment (marissaDubbelaar) [reply workflows/mhcquant.nf:196]: You are right, uncommented it

**Pull request merged**